### PR TITLE
Instal kraken monitor api

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: builder_from_package/debian10/Dockerfile-tyr-worker [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM navitia/master
 COPY navitia-common_*deb navitia-tyr_*.deb navitia-ed_*.deb navitia-cities_*.deb mimirsbrunn7_buster-*.deb cosmogony2cities_*.deb ./
 
 RUN apt-get update \
-    && apt install -y python python-pip git libgeos-c1v5 libpq5 tree\
+    && apt install -y python python-pip git libgeos-c1v5 libpq5 \
```
Comment (pbougue) [inline builder_from_package/debian10/Dockerfile-tyr-worker:6]: You could do the same for tyr-beat :wink:

File: builder_from_package/debian10/Dockerfile-kraken [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+    && apt install -y curl python python-pip \
+    && apt install -y ./navitia-monitor-kraken_*.deb \
     && apt install -y netcat --autoremove \
     && apt-get clean \
-    && rm -rf navitia-kraken_*.deb
+    && rm -rf navitia-kraken_*.deb \
+    && rm -rf navitia-monitor-kraken_*.deb \
+    && pip install --no-cache-dir -r /usr/share/monitor_kraken/requirements.txt
+
+COPY run_kraken.sh /usr/src/app/run.sh
```
Comment (pbougue) [inline builder_from_package/debian10/Dockerfile-kraken:16]: Where is that `run_kraken.sh`?

File: builder_from_package/debian8/Dockerfile-kraken [hunk 3]
```diff
@@ -2,22 +2,28 @@ FROM navitia/master
...
 RUN dpkg -i ./navitia-kraken_*.deb 
+RUN dpkg -i ./navitia-monitor-kraken_*.deb
 RUN apt-get install -f -y
 
-
 # put the binary in /srv/kraken
 RUN mkdir -p /srv/kraken
 RUN cp /usr/bin/kraken /srv/kraken/kraken
 
+RUN pip install --no-cache-dir -r /usr/share/monitor_kraken/requirements.txt
```
Comment (pbougue) [inline builder_from_package/debian8/Dockerfile-kraken:16]: Don't you need python and pip?

File: builder_from_package/debian10/Dockerfile-tyr-worker (hunk 1 above)
Comment (azime) [reply builder_from_package/debian10/Dockerfile-tyr-worker:6]: :+1:

File: builder_from_package/debian10/Dockerfile-kraken (hunk 2 above)
Comment (azime) [reply builder_from_package/debian10/Dockerfile-kraken:16]: OK

File: builder_from_package/debian8/Dockerfile-kraken (hunk 3 above)
Comment (azime) [reply builder_from_package/debian8/Dockerfile-kraken:16]: No, The _**master**_ image based on _**navitia/debian8_dev**_

**Review by woshilapin — APPROVED** [review]

**Review by pbougue — APPROVED** [review]
Not sure why the PR fails, but it looks good to me.

**Pull request merged**